### ISSUE TITLE
Multi-condition Info [igNotifier]

### ISSUE THREAD

**Issue by IG-Stanimir-Dimitrov** [IgniteUI/ignite-ui#1153]
### Description  
Enhance the igNotifier to support multi-condition info messages that notify the user if and what part of the conditions have been met.

**Comment by kdinev** [comment]
@damyanpetev Shouldn't this issue be closed as done?

**Comment by damyanpetev** [comment]
@kdinev Not quite, the current change for #1067 only creates multiple error messages (through the validator, notifier itself being completely unaware). What this describes would require the `igNotifier` to be able to show multiple errors and well as success messages at the same time as I see it.
